content/tutorials/mushr_unity_sim.md

```diff
@@ -0,0 +1,109 @@
+---
+title: "Using unity to simulate MuSHR"
+date: 2021-05-17T17:51:18+05:30
+summary: "Using a unity-based simulator in place of the default state prediction system that comes with mushr_base"
+difficulty: "Intermediate"
+duration: 0
+featured: false  # whether this is listed at / (must also be top 6 by weight). 
+active: true     # whether this is listed at /tutorials/
+draft: true      # whether Hugo considers this a draft
+weight: 3        # 2 = intro tutorial 3 = anything else
+---
+
+<h2> By: <a href=https://www.sidharthtalia.com/>Sidharth Talia</a></h2>
+
+<!-- Header figure required! -->
+<br>
+{{< figure src="/tutorials/mushr_unity_sim/thumbnail.gif" width="800" >}} <br>                           
```

Comment:
good gif, can you zoom and crop so it is only the rviz window?

Reply:
Will do.